config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -111,105 +116,6 @@ spec:
                     type: string
                   kafkaJvmPerfOpts:
                     type: string
-                  listenersConfig:
```

Comment:
is this removed on purpose?

Reply:
yes, this was part of the previous implementation. we don't use it anymore since listenersConfig contains info about the ports as well, which are required for the envoy deployment. In order to decouple the envoy deployment from the Kafka spec, I gave up on overriding `listenersConfig` at broker level and just added an override option for the `hostname.